src/main/resources/story/web/WalmartCart.story

```diff
@@ -0,0 +1,16 @@
+Meta:
+    ID_4
+
+Lifecycle:
+Examples:
+/tables/URL.table
+
+Scenario: Add to cart
+Given I am on a page with the URL '<URL_walmart>'
+When I reset context
+When I enter `car` in field located `By.xpath(//input[@aria-label='Search'])`
+When I click on element located `By.xpath(//button[@aria-label='Search icon'])`
+When I click on element located `By.xpath(//a[@link-identifier='107566679'])`
```

Comment:
Can this xpath be changed to a more stable one? Or this identifier can't change?